mhs/common/mhs_common/workflow/synchronous.py

```diff
@@ -84,8 +84,10 @@ def __init__(self,
             await wdo.set_outbound_status(wd.MessageStatus.OUTBOUND_MESSAGE_TRANSMISSION_FAILED)
             return 500, 'Error making outbound request', None
 
-        logger.info('0021', 'Response received from spine {startTime} {endTime}',
-                    {'startTime': start_time, 'endTime': end_time})
+        logger.audit('0101',
+                     'Outbound Synchronous workflow completed. Message sent to Spine and {Acknowledgment} received.',
+                     {'Acknowledgment': True})
```

Comment:
Optional: should this perhaps be
```suggestion
                     {'Acknowledgment': wd.MessageStatus.OUTBOUND_MESSAGE_RESPONSE_RECEIVED})
```